_readme/README.jl

```diff
@@ -4,63 +4,131 @@
-# ![](_readme/images/arrows_3d_Vec3d_Fancy.png)
+
+# ### 2D Plots
+
+# Then, let us consider a simple, 2D simplexified cartesian mesh Ω
+
+domain = (0, 1, 0, 1)
+cell_nums = (10, 10)
+model = CartesianDiscreteModel(domain, cell_nums) |> simplexify
+Ω = Triangulation(model)
+
+
+# The visualization of the vertices and edges of Ω can be achieved as follows
+
+fig = plot(Ω)
+wireframe!(Ω, color=:black, linewidth=2)
+scatter!(Ω, marker=:star8, markersize=20, color=:blue)
+
+save("images/2d_Fig1.png", fig) #!md
+# ![](_readme/images/2d_Fig1.png)
+
+# We now consider the nodal field uh
+
+reffe = ReferenceFE(lagrangian, Float64, 1)
+V = FESpace(model, reffe)
+uh = interpolate(x->sin(π*(x[1]+x[2])), V)
+
+
+# and plot it over Ω, adding a colorbar
+
+fig, _ , plt = plot(Ω, uh)
+Colorbar(fig[1,2], plt)
+
+save("images/2d_Fig11.png", fig) #!md
+# ![](_readme/images/2d_Fig11.png)
+
+# On the other hand, we may as well plot a cell field
+
+celldata = π*rand(num_cells(Ω)) .-1
+fig, _ , plt = plot(Ω, color=celldata, colormap=:heat)
+Colorbar(fig[2,1], plt, vertical=false)
+
+save("images/2d_Fig13.png", fig) #!md
+# ![](_readme/images/2d_Fig13.png)
+
+# If we are only interested in the boundary of Ω
+
+Γ = BoundaryTriangulation(model)
+fig, _ , plt = plot(Γ, uh, colormap=:algae, linewidth=10)
+Colorbar(fig[1,2], plt)
+
+save("images/2d_Fig111.png", fig) #!md
+# ![](_readme/images/2d_Fig111.png)
+
+# ### 3D Plots
+
+# In addition to the 2D plots, GridapMakie is able to handle more complex geometries. If we
+# take the mesh from the [first Gridap tutorial](https://gridap.github.io/Tutorials/stable/pages/t001_poisson/#Tutorial-1:-Poisson-equation-1)
+
+model = DiscreteModelFromFile("model.json")
+Ω = Triangulation(model)
+∂Ω = BoundaryTriangulation(model)
+fig = plot(Ω, shading=true)
+wireframe!(∂Ω, color=:black)
+save("images/3d_Fig1.png", fig) #!md
+# ![](_readme/images/3d_Fig1.png)
+
+v(x) = sin(π*(x[1]+x[2]+x[3]))
+fig, ax, plt = plot(Ω, v, shading=true)
+Colorbar(fig[1,2], plt)
+
+save("images/3d_Fig3.png", fig) #!md
+# ![](_readme/images/3d_Fig3.png)
+
+# we can even plot FE approximations in certain subdomains, e.g.
+
+Γ = BoundaryTriangulation(model, tags=["square", "triangle", "circle"])
+fig = plot(Γ, v, colormap=:rainbow, shading=true)
+wireframe!(∂Ω, linewidth=0.5, color=:gray)
+
+save("images/3d_Fig2.png", fig) #!md
+# ![](_readme/images/3d_Fig2.png)
+
+# ### Animations and interactivity
+
+# Finally, by using Makie [Observables](https://makie.juliaplots.org/stable/interaction/nodes.html), we
+# can create animations or interactive plots. For example, if the nodal field has a time dependence
+
+t = Observable(0.0)
+u = lift(t) do t
+    x->sin(π*(x[1]+x[2]+x[3]))*cos(π*t)
+end
+fig = plot(Ω, u, colormap=:rainbow, shading=true, colorrange=(-1,1))
+wireframe!(∂Ω, color=:black, linewidth=0.5)
+framerate = 30
+timestamps = range(0, 2, step=1/framerate)
+record(fig, "images/animation.gif", timestamps; framerate=framerate) do this_t
```

Comment:
I think this line of code assumes that a folder `images` is available at the current dir of execution. If the user executes this script away from the dir of the readme file `_readme`, it will likely fail. I would either add a command to create the `images` folder or store it simply at `animation.gif`, which should work.

Reply:
Or add `mkpath("images")` before recording.

Reply:
I think the best way is to create the `images` folder and store the figures and animations there, so that it's more comfortable for the user.

Reply:
Agreed